lib/omise/OmiseCharge.php

```diff
@@ -97,11 +97,16 @@ public function expire()
     /**
      * Captures a charge.
      *
+     * @param int|null $captureAmount  optional capture amount.
      * @return OmiseCharge
      */
-    public function capture()
+    public function capture(?int $captureAmount = null)
```

Comment:
Let's not add `type` here because it will cause errors for old PHP versions. We want our library to support from PHP 5.6 to latest version.

Reply:
It's better to expect `$params` instead of a field. You can see `refund()` method on how it's done. Since PHP is dynamic, doing this way will allow us to handle any parameters in the future without modifying code.

Reply:
@aashishgurung but capture expects only one parameter which I don't see changing in the near future

Reply:
I understand it but it's about using the flexibility given by the language to scale the library. If you see `refund()` or `charge()` methods then the way it is coded allows it to handle new parameters without breaking existing flow.

Reply:
ok got it